[2024-03-10T20:47:09.951+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: EnergyWarehouse.load_conso_energy_region manual__2024-03-10T20:46:17.709120+00:00 [queued]>
[2024-03-10T20:47:10.086+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: EnergyWarehouse.load_conso_energy_region manual__2024-03-10T20:46:17.709120+00:00 [queued]>
[2024-03-10T20:47:10.098+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 2
[2024-03-10T20:47:10.226+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): load_conso_energy_region> on 2024-03-10 20:46:17.709120+00:00
[2024-03-10T20:47:10.330+0000] {standard_task_runner.py:57} INFO - Started process 2213 to run task
[2024-03-10T20:47:10.368+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'EnergyWarehouse', 'load_conso_energy_region', 'manual__2024-03-10T20:46:17.709120+00:00', '--job-id', '935', '--raw', '--subdir', 'DAGS_FOLDER/***-to-bigquery-dag/***_to_bigquerry_dag.py', '--cfg-path', '/tmp/tmp3789rdl9']
[2024-03-10T20:47:10.388+0000] {standard_task_runner.py:85} INFO - Job 935: Subtask load_conso_energy_region
[2024-03-10T20:47:11.571+0000] {task_command.py:410} INFO - Running <TaskInstance: EnergyWarehouse.load_conso_energy_region manual__2024-03-10T20:46:17.709120+00:00 [running]> on host 952b3f8a1a94
[2024-03-10T20:47:12.442+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Mbaye SENE' AIRFLOW_CTX_DAG_ID='EnergyWarehouse' AIRFLOW_CTX_TASK_ID='load_conso_energy_region' AIRFLOW_CTX_EXECUTION_DATE='2024-03-10T20:46:17.709120+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-03-10T20:46:17.709120+00:00'
[2024-03-10T20:47:12.498+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/operators/python.py:198: FutureWarning: The error_bad_lines argument has been deprecated and will be removed in a future version.


  return self.python_callable(*self.op_args, **self.op_kwargs)

[2024-03-10T20:47:22.505+0000] {python.py:183} INFO - Done. Returned value was: None
[2024-03-10T20:47:22.568+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=EnergyWarehouse, task_id=load_conso_energy_region, execution_date=20240310T204617, start_date=20240310T204709, end_date=20240310T204722
[2024-03-10T20:47:22.970+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2024-03-10T20:47:23.713+0000] {taskinstance.py:2651} INFO - 1 downstream tasks scheduled from follow-on schedule check
